graphql/schemabuilder/reflect.go

```diff
@@ -49,6 +49,10 @@ type graphQLFieldInfo struct {
 
 	// KeyField indicates that this field should be treated as a Object Key field.
 	KeyField bool
+
+	// OptionalArg indicates that this field should be treated as an optional
+	// field on graphQL input args.
+	OptionalArg bool
```

Comment:
nit: how about `OptionalInputField` to match the language graphql uses to refer to this

Reply:
Done!